### DISCUSSION TITLE
Support for React Native?

### DISCUSSION THREAD

**Discussion by jolic** [aralroca/fragstore#56] [Ideas]
It would be nice if this tool could be used with React Native. 

Is that planed?

**Comment by aralroca** [comment]
Yes, is in the plan for version 1.0. 😊

In fact, with the current version I haven't even tested it yet. Have you been able to verify that it doesn't work in React Native? 

Thank you!

**Reply by jolic** [reply]
No, currently not. It would be nice that the store could be accessed also when we use RNN (React Native Navigation) as this is a kind of tricky (aka used Proxies).

**Comment by t1mofe1** [comment]
Hey! Have someone tested Teaful with RN/Expo?

**Reply by aralroca** [reply]
Not yet! The idea is to support it for version 1.
